aura-borealis-flask-app/app.py

```diff
@@ -18,13 +18,16 @@
 from live_data import get_all_warnings_counts_x
```

Comment:
@kdobolyi, when importing multiple functions from the same module in Python, I find [this syntax](https://github.com/IQTLabs/GitGeo/blob/d1ce3361dd279a29bf75687b4bb54145eaa93caa/test_gitgeo.py#L15) more concise. Consider refactoring.

Reply:
done!